dev-requirements.txt

```diff
@@ -7,3 +7,4 @@ mock
 requests-mock
 pynsist
 unittest2; python_version < '2.7'
+versioneer
```

Comment:
installing `versioneer` on travis or locally is not necessary to build streamlink,
because `versioneer.py` is in the main folder for `setup.py`

so this could be removed

Reply:
True :)